addons/web/static/tests/webclient/actions/client_action.test.js

```diff
@@ -156,6 +156,20 @@ test("soft_reload will refresh data", async () => {
     expect.verifySteps(["web_search_read"]);
 });
 
+test("soft_reload a form view", async () => {
+    onRpc("web_read", ({ args }) => {
+        expect.step(`read ${args[0][0]}`);
+    });
+    await mountWithCleanup(WebClient);
+    await getService("action").doAction(3);
+    await contains(".o_data_row .o_data_cell").click();
+    await contains(".o_pager_next").click();
```

Comment:
`.o_form_view .o_pager_next` to make it explicit and to avoid future hypothetical undeterminism at least on copy/paste